tests/fatrace-json

```diff
@@ -1,156 +0,0 @@
-#!/bin/sh
-set -euC
-
-mkdir -m 777 tmp
-trap "rm -rf tmp" EXIT INT QUIT PIPE
-
-LOG="$AUTOPKGTEST_TMP/fatrace.log"
-echo "starting fatrace..."
-fatrace --current-mount --user -s 10 -o $LOG --json &
-sleep 1
-
-# path
-touch "tmp/1-good.tmp"
-
-# path_raw
-touch "tmp/2-bad-$(printf '\001').tmp"
-
-# uid and gid
-runuser -u nobody touch "tmp/3-good.tmp"
```

Comment:
Dropped this, as test_user() covers this already, and it requires some special handling.